crates/control-interface/src/broker.rs

```diff
@@ -11,10 +10,6 @@ fn prefix(topic_prefix: &Option<String>, lattice_prefix: &str) -> String {
     )
 }
 
-pub fn control_event(lattice_prefix: &str) -> String {
```

Comment:
This was only used in one place, so I inlined it to prevent more usage before we remove this subject